shell.nix

```diff
@@ -0,0 +1,12 @@
+{ pkgs ? import (fetchTarball "https://github.com/NixOS/nixpkgs/archive/refs/tags/23.05.tar.gz") {} }:
+
+pkgs.mkShell {
+  packages = with pkgs; [
+    curl
+    git
+    python39
+    clang
```

Comment:
OK, with `hdrhistogram` officially publishing wheels, clang shouldn't be necessary anymore